src/intl/en/page-learn.json

```
@@ -1,37 +1,37 @@
   "hero-subtitle": "Your educational guide to the world of Ethereum. Learn how Ethereum works and how to connect to it. This page includes technical and non-technical articles, guides, and resources.",
   "hero-button-lets-get-started": "Let's get started",
-  "what-is-crypto-1": "You may have heard about cryptocurrencies, Bitcoin, and blockchain. This page will help you learn about what they are how they relate to Ethereum.",
-  "what-is-crypto-link-1": "And what is Ethereum anyway?",
-  "what-is-crypto-2": "Bitcoin enables anyone to transfer money globally. Ethereum does too, <b>and</b> it enables anyone to create and agree on any kind of contract they can imagine. It’s both resilient and flexible: any computer program can run on Ethereum.",
+  "what-is-crypto-1": "You may have heard about cryptocurrencies, blockchains and Bitcoin. The links below will help you to learn what they are and how they relate to Ethereum.",
+  "what-is-crypto-2": "Cryptocurrencies, such as Bitcoin, enable anyone to transfer money globally. Ethereum does too, but it can also run code that enables people to create apps and organizations. It’s both resilient and flexible: any computer program can run on Ethereum. Learn more and find out how to get started:",
   "what-is-ethereum-card-title": "What is Ethereum?",
   "what-is-ethereum-card-description": "If you are new, start here to learn why Ethereum matters.",
   "what-is-ethereum-card-image-alt": "Illustration of a person peering into a bazaar, meant to represent Ethereum.",
   "what-is-eth-card-title": "What is ETH?",
   "what-is-eth-description": "Ether (ETH) is the currency powering the Ethereum network and apps.",
-  "where-can-i-get-eth-card-title": "Where can I get ETH?",
-  "where-can-i-get-eth-card-description": "There are many ways to get ETH, depending on your location.",
+  "what-is-web3-card-title": "What is Web3?",
+  "what-is-web3-card-description": "Web3 is a model for the internet valuing ownership of your own assets and identity.",
   "additional-reading-more-on-ethereum-basics": "More on Ethereum basics",
+  "guides-hub-desc": "Guides: step-by-step instructions on how to use Ethereum",
+  "quiz-hub-desc": "Quiz hub: test your knowledge and learn more",
   "additional-reading-what-are-smart-contracts": "What are smart contracts?",
-  "additional-reading-a-developers-intro": "An introduction to Ethereum for developers",
   "additional-reading-what-is-web3": "What is web3?",
-  "additional-reading-value": "What is value?",
-  "additional-reading-decentralize-everything": "Decentralize everything",
-  "how-do-i-use-ethereum-1": "Using Ethereum is like moving to a new place. Things will seem unfamiliar at first, and everyone moves for their own reasons. Despite this, the first thing you'll need are new keys. The software that creates and holds the keys to your new home is called a \"wallet\".",
+  "additional-reading-decentralize-everything": "Video: Explanation of what Ethereum is by Vitalik Buterin",
```

I'd use [this devcon talk](https://www.youtube.com/watch?v=UihMqcj-cqc) and change the copy here to 'Ethereum in 30 minutes - Vitalik Buterin'

cc: @konopkja 

```suggestion
  "additional-reading-decentralize-everything": "Video: Explanation of what Ethereum is by Vitalik Buterin",
```

---

```suggestion
  "additional-reading-decentralize-everything": "Ethereum in 30 minutes by Vitalik Buterin",
```